### PR TITLE
fix: expose aws-cfg-ephemeral for pulumi & terraform provider

### PR DESCRIPTION
**Pull request by raksiv**
Some services may require updating the default ephemeral storage setting, this PR exposes that as part of the Nitric yaml for the AWS Pulumi and Terraform providers.

### CODE REVIEW

**Review by tjholm — DISMISSED** [review]

**Comment by tjholm** [comment]
@raksiv can you add tf support for this as well.

**Review by tjholm — APPROVED** [review]

File: cloud/aws/deploytf/generated/service/ServiceConfig.go [hunk 1]
```diff
@@ -21,13 +21,15 @@ type ServiceConfig struct {
 	ServiceName *string `field:"required" json:"serviceName" yaml:"serviceName"`
 	// The ID of the Nitric stack.
 	StackId *string `field:"required" json:"stackId" yaml:"stackId"`
+	// The amount of ephemeral storage to allocate to the lambda function 512.
```
Comment (jyecusch) [inline cloud/aws/deploytf/generated/service/ServiceConfig.go:24]:
```suggestion
	// The amount of ephemeral storage to allocate to the lambda function in MB.
```

File: cloud/aws/deploytf/generated/service/ServiceConfig.go [hunk 2]
```diff
@@ -21,13 +21,15 @@ type ServiceConfig struct {
...
+	EphemeralStorage *float64 `field:"optional" json:"ephemeralStorage" yaml:"ephemeralStorage"`
 	// The amount of memory to allocate to the lambda function 128.
 	Memory *float64 `field:"optional" json:"memory" yaml:"memory"`
 	// The security group ids to use for the aws lambda function.
 	SecurityGroupIds *[]*string `field:"optional" json:"securityGroupIds" yaml:"securityGroupIds"`
 	// The subnet ids to use for the aws lambda function.
 	SubnetIds *[]*string `field:"optional" json:"subnetIds" yaml:"subnetIds"`
-	// The timeout for the lambda function 10.
+	// The timeout for the lambda function 15.
```
Comment (jyecusch) [inline cloud/aws/deploytf/generated/service/ServiceConfig.go:32]:
```suggestion
	// The timeout for the lambda function in seconds.
```

File: cloud/aws/deploytf/generated/service/ServiceConfig.go [hunk 3]
```diff
@@ -21,13 +21,15 @@ type ServiceConfig struct {
 	ServiceName *string `field:"required" json:"serviceName" yaml:"serviceName"`
 	// The ID of the Nitric stack.
 	StackId *string `field:"required" json:"stackId" yaml:"stackId"`
+	// The amount of ephemeral storage to allocate to the lambda function 512.
+	EphemeralStorage *float64 `field:"optional" json:"ephemeralStorage" yaml:"ephemeralStorage"`
 	// The amount of memory to allocate to the lambda function 128.
```
Comment (jyecusch) [inline cloud/aws/deploytf/generated/service/ServiceConfig.go:26]:
```suggestion
	// The amount of memory to allocate to the lambda function in MB.
```

**Review by jyecusch — DISMISSED** [review]

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**